.gitignore

```diff
@@ -37,3 +37,4 @@ bin/
 *.iml
 maven.deps
 DEPENDENCIES
+lib/
```

Comment:
Does this ignore all folders named `lib` of the entire reposistory? If you want to ignore a specific lib folder better add a new .gitignore file in its parent directory